devops/actions/llvm_test_suite/action.yml

```diff
@@ -81,7 +112,10 @@ runs:
       export PATH=$PWD/toolchain/bin/:$PATH
       # TODO make this part of container build
       export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/rocm/hip/lib/:/opt/rocm/lib
-      export LIT_OPTS="-v --no-progress-bar --show-unsupported --max-time 3600 --time-tests -o $PWD/build/results_${{ inputs.results_name_suffix }}.json"
+      # For standalone llvm-test-suite
+      export LIT_OPTS="-v --no-progress-bar --show-unsupported --max-time 3600 --time-tests -o $PWD/build/results_${{ inputs.results_name_suffix }}_${{ inputs.in_tree_e2e }}.json"
+      # For in-tree End-to-End tests
+      export SYCL_TEST_E2E_TARGETS="-v --no-progress-bar --show-unsupported --max-time 3600 --time-tests -o $PWD/build/results_${{ inputs.results_name_suffix }}_${{ inputs.in_tree_e2e }}.json"
```

Comment:
```suggestion
      export SYCL_TEST_LIT_OPTS="-v --no-progress-bar --show-unsupported --max-time 3600 --time-tests -o $PWD/build/results_${{ inputs.results_name_suffix }}_${{ inputs.in_tree_e2e }}.json"
```
?

Reply:
Good catch. The fix is to remove the second one alltogether. `LLVM_LIT` controls the behavior of the `llvm-lit` directly and not something in our CMake configurations.

Reply:
Cool. I thought that was some problem with using `LLVM_LIT` directly and you introduced a new option specifically to avoid some problems.

Is this the only place where is should be removed? Don't we process this variable in the lit python scripts?

Reply:
It was two issues in one place. First, unnecessary processing and second, wrong name altogether.

`SYCL_TEST_E2E_TARGETS` (the real one) is needed and used in the cmake invocation, replacing old `CHECK_SYCL_ALL` control. See

https://github.com/intel/llvm/pull/8081/files#diff-453e62b4f1f42795824f7739ee9c6b9f6eeec42cebbeb66968fb18af93bb07a1R101
https://github.com/intel/llvm/pull/8081/files#diff-accae39591e7e6aec1708426e892c3fc128b4b0fd3cae2a332752eff51374d3fR42

There is also `SYCL_E2E_TESTS_LIT_FLAGS` replacing `TEST_SUITE_LIT_FLAGS` used in

https://github.com/intel/llvm/pull/8081/files#diff-accae39591e7e6aec1708426e892c3fc128b4b0fd3cae2a332752eff51374d3fR59